build/serve.js

```diff
@@ -102,7 +102,7 @@ gulp.task('serve:watch', ['watch'], serveDevelopmentMode);
 /**
```

Comment:
Also, 'spawn-backend' hasnt been added before, because it does not work on Travis. I.e., integration-test:prod requires serve:prod, which in order to work on Travis needs Kubernetes apiserver host. We need development Kubernetes instance first to enable this.

Does this make sense?